Packs/XSOARFileManagement/Integrations/XSOARFileManagement/XSOARFileManagement.py

```diff
@@ -339,32 +368,49 @@ def upload_file_command(client: Client, args: dict) -> CommandResults:
     """
     incident_id = args.get('incidentID', demisto.incident()["id"])
     file_content = args.get('fileContent', '')
+    file_content_b64 = args.get('fileContentB64', '')
     entry_id = args.get('entryID', '')
     file_path = args.get('filePath', '')
     file_name = args.get('fileName', '')
     target = args.get('target', 'war room entry')
 
+    # id can be empty if this command is triggered by a field changed
+    if not incident_id:
+        return_error("Unable to get the incident id of the incident !")
     # check if some content is given and not too many
-    if len(list(filter(None, [file_content, entry_id, file_path]))) != 1:
-        return_error("You have to give either the content of the file using the arg 'fileContent' or an entryID or a file path !")
+    if len(list(filter(None, [file_content, file_content_b64, entry_id, file_path]))) != 1:
+        return_error("You have to give either the content of the file using the arg 'fileContent'"
+                     "or 'fileContentB64' or an entryID or a file path !")
     # if file_name is not set when using content of the file
-    if file_content and not file_name:
+    if (file_content or file_content_b64) and not file_name:
         return_error("You have to choose a name for your file !")
 
     response = {}
     if file_content:
-        response = client.upload_file(incident_id, file_content, file_name, target == 'incident attachment')
+        response = client.upload_file(incident_id,
+                                      file_content,
+                                      file_name,
+                                      target == 'incident attachment')
+    elif file_content_b64:
+        file_content_tmp = base64.b64decode(file_content_b64)
+        response = client.upload_file(incident_id,
+                                      file_content_tmp,
+                                      file_name,
+                                      target == 'incident attachment')
     else:
         arg_path: str = list(filter(None, [entry_id, file_path]))[0]
         res_path, res_name = get_file_path_name(arg_path)
         # file name override by user
         file_name = file_name if file_name else res_name
         if not file_name:
-            return_error("Impossible to detect a filename in the path, use the argument 'fileName' to set one !")
+            return_error("Impossible to detect a filename in the path,"
+                         "use the argument 'fileName' to set one !")
         file_binary = open(res_path, 'rb')
-        response = client.upload_file(incident_id, file_binary, file_name, target == 'incident attachment')
+        response = client.upload_file(incident_id,
+                                      file_binary,
+                                      file_name,
+                                      target == 'incident attachment')
         file_binary.close()
-    demisto.results(response)
```

Comment:
Why did you remove it?

Reply:
It outputs the result of the API and it's not useful, I missed it when I published the integration ^^"